pages/legal/privacy.js

```diff
@@ -15,25 +15,217 @@ export default withData(({url}) => {
+
+Solche auf freiwilliger Basis von einer betroffenen Person an die Verantwortlichen übermittelten personenbezogenen Daten werden für Zwecke der Bearbeitung oder der Kontaktaufnahme mit der betroffenen Person gespeichert. Es erfolgt keine Weitergabe dieser personenbezogenen Daten an Dritte.
+
+**7. Kommentar- und Debattierfunktion auf der Internetseite**
+
+Die Verantwortlichen bieten den Besucher/-innen der Internetseite www.republik.ch die Möglichkeit, individuelle Kommentare zu einzelnen Magazinbeiträgen zu hinterlassen sowie Debatten zu bestimmten Themen zu führen. Diese Kommentare können in der Regel von Dritten kommentiert werden.
+
+Hinterlässt eine betroffene Person einen Kommentar in dem auf dieser Internetseite veröffentlichten Artikel, werden neben dem von der betroffenen Person hinterlassenen Kommentar auch Angaben zum Zeitpunkt der Kommentareingabe sowie der Name (Klarnamen-Pflicht) der betroffenen Person gespeichert und veröffentlicht. Ferner wird die vom Internet-Service-Provider (ISP) der betroffenen Person vergebene IP-Adresse mitprotokolliert. Diese Speicherung der IP-Adresse erfolgt aus Sicherheitsgründen und für den Fall, dass die betroffene Person durch einen abgegebenen Kommentar die Rechte Dritter verletzt oder rechtswidrige Inhalte postet. Es erfolgt keine Weitergabe dieser erhobenen personenbezogenen Daten an Dritte, sofern eine solche Weitergabe nicht gesetzlich vorgeschrieben ist oder der Verteidigung der Rechte der Verantwortlichen dient.
+
+**8. Routinemässige Löschung und Sperrung von personenbezogenen Daten**
+
+Die Verantwortlichen verarbeiten und speichern personenbezogene Daten der betroffenen Person nur für den Zeitraum, der zur Erreichung des Speicherungszwecks erforderlich ist, oder sofern dies durch gesetzliche Vorgaben, welchen die Verantwortlichen unterliegen, vorgesehen wurde.
+
+Entfällt der Speicherungszweck oder läuft eine allfällige gesetzlich gesetzte Speicherfrist ab, werden die personenbezogenen Daten routinemässig und entsprechend den gesetzlichen Vorschriften gesperrt oder gelöscht.
+
+**9. Rechtsgrundlage der Verarbeitung**
+
+Es werden keine Daten ohne gesetzliche Grundlage bearbeitet. Als Rechtsgrundlage dienen 
+
+- die Einwilligung,
+- die Erfüllung eines Vertrages oder vorvertraglicher Massnahmen,
+- die rechtlichen Verpflichtungen oder der Schutz von lebenswichtigen Interessen der betroffenen Person oder einer anderen natürlichen Person,
+- die Wahrung eines berechtigten Interesses der Verantwortlichen (z. B. wirtschaftliche Interessen) oder eines Dritten.
+
+**10. Rechte der betroffenen Person**
+
+a)    Recht auf Bestätigung
+
+Jede betroffene Person hat das Recht, von den Verantwortlichen eine Bestätigung darüber zu verlangen, ob sie betreffende personenbezogene Daten verarbeitet werden. Möchte eine betroffene Person dieses Bestätigungsrecht in Anspruch nehmen, kann sie sich hierzu jederzeit an die Mitarbeiter/-innen der Verantwortlichen wenden. 
+
+b)    Recht auf Auskunft
```

Comment:
Let's get rid of the extra spaces in
`i)______Recht auf`
(in several other places here as well)